app/models/solidus_pay_tomorrow/gateway.rb

```diff
@@ -12,8 +12,17 @@ def purchase
       not_implemented(__method__)
     end
 
-    def capture
-      not_implemented(__method__)
+    def capture(gateway_options:)
+      capture_response = SolidusPayTomorrow::Client::CaptureService.call(
+        order_token: gateway_options[:order_token],
+        payment_method: gateway_options[:payment_method]
+      )
+      ActiveMerchant::Billing::Response.new(
+        true,
+        'Transaction Captured', capture_response
```

Comment:
Please return the Application token here so solidus will automatically create the payment with the right response code

Reply:
Since we already create the Payment early on, is this needed?
When this code will be executing, we'd already have a `Spree::Payment` object in `pending` state

Reply:
Added